services/movie.py

```diff
@@ -0,0 +1,40 @@
+from db.models import Movie
+
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet[Movie] | Movie:
```

Comment:
The return type hint `QuerySet[Movie] | Movie` is incorrect. Since `get_movies` always returns a QuerySet, it should be `QuerySet[Movie]`.